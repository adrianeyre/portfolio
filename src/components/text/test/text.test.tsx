import React from 'react';
import { shallow } from 'enzyme';

import ITextProps from '../interface/text-props';
import Text from '../text';

describe('Text', () => {
	const defaultProps: ITextProps = {
		data: undefined,
	};
	
	it('Should render correctly', () => {
		const login = shallow(<Text { ...defaultProps }/>);

		expect(login).toMatchSnapshot();
	});
});