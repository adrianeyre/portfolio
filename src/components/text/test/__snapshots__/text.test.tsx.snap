// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Should render correctly 1`] = `ShallowWrapper {}`;
